boards/arm/degu_evk/Kconfig.defconfig

```diff
@@ -0,0 +1,103 @@
+# Kconfig - Degu Evaluation Kit configuration
+#
+# Copyright (c) 2019 Atmark Techno, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+if BOARD_DEGU_EVK
+
+config BOARD
+	default "degu_evk"
+
+if USB
+
+config USB_NRF52840
+	def_bool y
+
+config USB_DEVICE_STACK
+	def_bool y
+
+endif # USB
+
+if IEEE802154
+
+config IEEE802154_NRF5
+	def_bool y
+
+endif # IEEE802154
+
+config I2C
+	def_bool y
```

Comment:
This option is assigned the default value of `y` in [degu_evk_defconfig](https://github.com/zephyrproject-rtos/zephyr/pull/18796/files#diff-cff9f6f0a5f3e232384139980629c7c0R23), no need to repeat it here.